packages/actor-rdf-parse-html/components/Actor/RdfParse/Html.jsonld

```diff
@@ -0,0 +1,49 @@
+{
+  "@context": [
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-abstract-mediatyped/^1.0.0/components/context.jsonld",
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-rdf-parse-html/^1.0.0/components/context.jsonld",
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/bus-rdf-parse/^1.0.0/components/context.jsonld"
+  ],
+  "@id": "npmd:@comunica/actor-rdf-parse-html",
+  "components": [
+    {
+      "@id": "carph:Actor/RdfParse/Html",
+      "@type": "Class",
+      "extends": "cbrp:Actor/RdfParseFixedMediaTypes",
+      "requireElement": "ActorRdfParseHtml",
+      "comment": "An HTML actor that listens on the 'rdf-parse' bus.",
+      "parameters": [
+        {
+          "@id": "carph:mediatorRdfParse",
+          "comment": "The RDF Parse mediator",
+          "required": true,
+          "unique": true
+        },
+        {
+          "@id": "caam:Actor/AbstractMediaTypedFixed/mediaTypeEntry",
+          "defaultScoped": {
+            "defaultScope": "carph:Actor/RdfParse/Html",
+            "defaultScopedValue": [
+              { "mediaTypeKey": "text/html", "priorityValue": "1.0" },
+              { "mediaTypeKey": "(text/html; charset=utf-8)", "priorityValue": "0.8" },
+              { "mediaTypeKey": "(text/html; charset=utf-8)", "priorityValue": "0.9" },
+              { "mediaTypeKey": "text/html; charset=utf-8", "priorityValue": "1.0" },
+              { "mediaTypeKey": "hypermedia-html", "priorityValue": "0.7" }
```

Comment:
I don't think this is a standardized media type (correct me if I'm wrong).

Reply:
I totally forgot about those. text/html should indeed be sufficient